### PR TITLE
JTJ-59-Goal-Delete

### PR DESCRIPTION
**Pull request by tgi-wtag**
Tiny PR, just the Delete journal Entry Method, i added no video because i believe these few lines of code are explanation enough. thank you anyways for reviewing 🌻

### CODE REVIEW

File: app/controllers/goals_controller.rb [hunk 1]
```diff
@@ -28,6 +28,11 @@ def update
     end
   end
 
+  def destroy
+    @goal.destroy
+    redirect_to user_goals_path(@user)
```
Comment (chm-wtag) [inline app/controllers/goals_controller.rb:33]: what happens if destroy operation fails? i think we should handle that case as well

Comment (chm-wtag) [reply app/controllers/goals_controller.rb:33]: also we can add authorization to check whether the current user is allowed to destroy the goal or not

Comment (chm-wtag) [reply app/controllers/goals_controller.rb:33]: add proper tests after addressing the above mentioned comments

**Review by chm-wtag — APPROVED** [review]

**Review by evk-wtag — APPROVED** [review]

File: app/controllers/goals_controller.rb (hunk 1 above)
Comment (tgi-wtag) [reply app/controllers/goals_controller.rb:33]: thank you i added the edge case and the spec regarding it

**Review by lra-wtag — APPROVED** [review]

**Pull request merged**